lib/src/code_field/code_field.dart

```diff
@@ -238,6 +237,13 @@ class _CodeFieldState extends State<CodeField> {
     super.dispose();
   }
 
+  @override
+  void didUpdateWidget(covariant CodeField oldWidget) {
```

Comment:
Should here also be `_onTextChanged` and `_updatePopupOffset`?

Reply:
added